test/diff.jl

```diff
@@ -33,6 +33,36 @@ dcsch = D(csch(t))
 @test isequal(expand_derivatives(D2(t)), 0)
 @test isequal(expand_derivatives(D2(5)), 0)
 
+# Vector calculus operators
+@variables vv(t,x,y)
+∇z = Gradient(z)
+∇xy = Gradient(x,y)
```

Comment:
what does this one mean?

Reply:
Gradient in the x and y dimensions. If x and y are in a cartesian domain this would be equivalent to a symbolic array `[d/dx,d/dy]`. But if x and y are polar coordinates (r and theta) this would be `[d/dx,1/x*d/dy]`. So I was thinking we would leave in symbolic `Gradient` form until the discretization, when the domain has been specified.